### ISSUE TITLE
AOSP 11.0 v300.j system partion is read only

### ISSUE THREAD

**Issue by mrksbk** [phhusson/treble_experimentations#1613]
hello,
first of all thanks for your hard work to bring us a aosp image with gapps included.
the image works very well except the issue that the system partion is read only so it`s not possible to overwirte files on the system partition. adaway for example does not work.

i have installed "system-roar-arm64-ab-gapps.img.xz" on my essential ph-1.

best regards,
markus

**Comment by phhusson** [comment]
Yes it's on purpose, see ChangeLog. You can flash vndklite variant instead.

**Comment by mrksbk** [comment]
Hello,
thank you very much for your quick response. i really appreciate it.
i just flashed the vdnklite version of your provided system image and it works flawlessly.

best regards,
markus
